lib/publiccloud/provider.pm

```diff
@@ -370,7 +370,7 @@ sub terraform_apply {
         $cmd .= "-var 'name=" . $name . "' ";
         $cmd .= "-var 'project=" . $args{project} . "' " if $args{project};
         $cmd .= "-var 'enable_confidential_vm=true' "    if $args{confidential_compute};
-        $cmd .= sprintf(q(-var 'tags={"openqa_ttl":"%d"}' ), get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300));
+        $cmd .= sprintf(q(-var 'tags=%s' ), escape_single_quote($self->terraform_param_tags));
```

Comment:
```suggestion
        $cmd .= sprintf(q(-var 'tags=%s' ), $self->terraform_param_tags());
```

Reply:
as mentioned [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13245#discussion_r707448166) I think it is wrong.